docs/source/whatsnew/version3.rst

```diff
@@ -217,6 +217,12 @@ Backwards incompatible changes
     to read and write notebook files
     instead of the deprecated :mod:`IPython.nbformat.current` APIs.
 
+  You can downgrade a notebook to v3 via ``nbconvert``::
+  
+      ipython nbconvert --to notebook --nbformat 3 [notebook]
```

Comment:
`<notebook>` ? isn't `[]` for optional and `<>` for required usually ? That's my feeling, but I'm not sure.